runserver.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
 pip install -r requirements.txt
+service cron start
```

Comment:
This appears to be putting a system-level assumption (cron service, runnable by `$(whoami)`, not already running) in an app-specific script. Is there a way to move this into docker config?